docs/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our **CONTRIBUTING** document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+
+<!-- These have been added according to the rOpenSci guidelines. -->
+<!-- From: https://www.talater.com/open-source-templates/#/page/99 -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue. List the bug ID if applicable)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing core functionality to change) 
+
+
+### New Feature Submissions checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+1. [ ] Have you ensured there are no conflicts with major packages (e.g. tidyverse, MASS) or base R packages?
+2. [ ] Does your new feature require documentation, and if so, have you udpated the documentation?
+3. [ ] Have you added tests for your features, and do your features pass all tests?
+4. [ ] Have you linted your code with lintr locally prior to submission?
+5. [ ] Have you styled your code with styler locally prior to submission?
+6. [ ] Have you checked the performance of your code?
+7. [ ] Does your code comply with rOpenSci guidelines?
+8. [ ] Does your code comply with CRAN policies?
+
+### Changes to Core Features:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
```

Comment:
Do we need to add something about how backwards compatibility has been addressed? Or is this meant to be satisfied by tests?

Reply:
Backward compatibility is a CRAN requirement, should we simply refer back to that?

Reply:
> Do we need to add something about how backwards compatibility has been addressed? Or is this meant to be satisfied by tests?

Yes, we could also simply require a standard backward compatibility test

Reply:
If tests exist that define what the API should do, then breaking changes should involve changing existing tests, so this should certainly be identified during a code review.

You can ask users to note if they think their code breaks backwards compatibility (it sort of does above).

Reply:
You can ask contributors for 1) backwards compatibility and 2) compatibility with the current development version. Tests to check for compatibility will also be implemented in the workflow, however, unless you want to spend time figuring out why checks for new functionality have failed, it's good to ask contributors to do it themselves prior to making a PR.

Reply:
How about simply an additional question: "Have you demonstrated backwards compatibility and compatibility with the current development version, or discussed the break in backwards compatibility?"

Reply:
Addressed by https://github.com/josephguillaume/hydromad/pull/50/commits/be2943aba68636a567ad56c9a0c2bebca6fa6896